src/csharp/Grpc.Tools.MSBuildIntegration/Grpc.Tools.MSBuildIntegration.targets

```diff
@@ -0,0 +1,27 @@
+<Project xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
+  <PropertyGroup>
+    <CoreCompileDependsOn>Proto;$(CoreCompileDependsOn)</CoreCompileDependsOn>
+  </PropertyGroup>
+  <ItemGroup>
+    <AvailableItemName Include="Proto"/>
+  </ItemGroup>
+  <Target Name="Proto" Inputs="@(Proto)" Outputs="@(Proto->'$(IntermediateOutputPath)%(FileName).cs')">

```

Comment:
Ah, I see. I tested with many files but none had an underscore in their name. Can you think of any other naming changes so I can modify the task/CSharpGenerator to work correctly? _Now I'm glad I have the C# based Grpc.Tools.CSharpGenerator to work these things out with_